src/evilHangman/evilHangmanGame.ts

```diff
@@ -0,0 +1,163 @@
+import fs from 'fs';
+import { isNonEmptyArray } from '../helpers/guards/isNonEmptyArray';
+
+const DICTIONARY_PATH = './res/dictionary.txt';
+const allWords = fs
+	.readFileSync(DICTIONARY_PATH)
+	.toString()
+	.split('\n')
+	.map(word => word.trim());
```

Comment:
Can we move this dictionary generation code into a separate file, `evilHangmanDictionary.ts` or the like?

Reply:
done